packages/block-library/src/pullquote/style.scss

```diff
@@ -1,21 +1,29 @@
-.wp-block-pullquote {
+figure.wp-block-pullquote {
```

Comment:
Is the `figure` really necessary? I think we should be striving for the weakest possible selectors for applying base styles to blocks.

Reply:
Thank you for your inputs @ZebulanStanphill!
In this case, figure was added to the selector on purpose to increase the specificity of our rule, without it the rules are not used.